src/Components/Aspire.RabbitMQ.Client/RabbitMQClientSettings.cs

```diff
@@ -34,4 +34,16 @@ public sealed class RabbitMQClientSettings
     /// The default value is <see langword="false"/>.
     /// </value>
     public bool DisableTracing { get; set; }
+
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether auto activation is disabled or not.
+    /// </summary>
+    /// <remarks>
+    /// When auto activation is enabled (the default), the RabbitMQ connection is established at startup time rather than on first use,
+    /// which prevents blocking threads when the connection is first requested from the DI container.
+    /// </remarks>
+    /// <value>
+    /// The default value is <see langword="false"/>.
+    /// </value>
+    public bool DisableAutoActivation { get; set; }
```

Comment:
@davidfowl @DamianEdwards @sebastienros -

What do you think about making this default to `true` (i.e. Disabled) for Aspire 9.5. And then in 10, we flip the switch to make it enabled by default? That would be less risky, and would allow us to test it in more scenarios.

That way if this breaks something, it won't be on by default.

Reply:
Ok, I did this. I kept the property named the same `DisableAutoActivation` but defaulted it to `true`. The intention is that in the next major version we can change the default to `false`.